.github/actions/get-keycloak-url/action.yml

```diff
@@ -14,11 +14,6 @@ runs:
       run: |
         KEYCLOAK_URL=https://$(kubectl get routes -n "${{ inputs.project }}" -l app=keycloak -o jsonpath='{.items[*].spec.host}')
         echo "KEYCLOAK_URL=$KEYCLOAK_URL" >> "$GITHUB_ENV"
-    - id: get-keycloak-site-url
```

Comment:
moving the fetching of site url portion out of this action, as we would need to conditionally fetch this based on the LB setup.